### DISCUSSION TITLE
input a newline in the input box

### DISCUSSION THREAD

**Discussion by sandycs-protoss** [open-webui/open-webui#2708] [General]
# Bug Report
How can we input a newline in the input box? It seems that SHIFT+ENTER or ALT+ENTER both don't work, they just immediately send the content.
Reproduction Steps:

## Description

**Steps to Reproduce:**
1. Copy a piece of text into the input box.
2. At the end of this text, press: Shift+Enter.

**Expected Behavior:**
After shift+Enter, it should be add a newline in the input box
**Actual Behavior:**
 immediately send the content.

## Environment

- **Open WebUI Version:** [v0.1.125]
- **Operating System:** [e.g., Windows 11]

**Comment by Z-nonymous** [comment]
FYI
Since then Shift+Enter now works,
https://deepwiki.com/search/how-can-i-enter-a-new-line-whe_71d536d3-9477-47fa-8492-0afd9aa80884?mode=fast

Nowadays it can be faster to ask AI than ask someone else, it's even faster than looking at the code yourself.
